<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Detenido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="statusButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>681</width>
     <height>521</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>31</width>
       <height>161</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>31</width>
       <height>161</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDigitalGauge" name="qdigitalgauge">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="Counter" name="counter">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
    <widget class="QwtCompass" name="Compass">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>0.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>-20.000000000000000</double>
     </property>
     <property name="scaleMaxMajor">
      <number>20</number>
     </property>
     <property name="scaleStepSize">
      <double>0.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
     <property name="singleSteps">
      <UInt>1</UInt>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="minScaleArc">
      <double>0.000000000000000</double>
     </property>
     <property name="maxScaleArc">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCnx">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Conexi√≥n USB</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="serialPortLabel">
     <property name="text">
      <string>Puerto Serie:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serialPortComboBox"/>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pingButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Counter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/counter.h</header>
  </customwidget>
  <customwidget>
   <class>QDigitalGauge</class>
   <extends>QWidget</extends>
   <header>qdigitalgauge.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
